front/admin/db.ts

```diff
@@ -67,6 +67,7 @@ import { DustAppSecret } from "@app/lib/models/dust_app_secret";
 import { ExtensionConfigurationModel } from "@app/lib/models/extension";
 import { FeatureFlag } from "@app/lib/models/feature_flag";
 import { MembershipInvitation } from "@app/lib/models/membership_invitation";
+import { PersonalDataSourceConnection } from "@app/lib/models/personal_data_source_connection";
```

Comment:
nit: should we call it `LabsPersonalDataSourceConnection` ?

Reply:
even the table ?